csss-site/src/events/templates/events/frosh/2020/index.html

```diff
@@ -0,0 +1,126 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Frosh Week 2020" />
+    <meta name="author" content="Jace Manshadi" />

```

Comment:
I am certainly not the author

Reply:
I only changed the Image on the page, so in a way, you kind of are.

Reply:
the name in this context is the person who updated the website for the current frosh year, which is you.